search-sentiments-map/routes/trends.js

```diff
@@ -74,10 +72,16 @@ async function updateDailyTrends() {
   saveTrendsAndDeletePrevious(trendsByCountry);
 }
 
-/** Creates a JSON item for trends in the given country. */
+/** 
+ * Creates a JSON item for trends in the given country.
+ * @param {?Array<JSON>} trendingSearches The JSON array including the top trends 
```

Comment:
"The JSON array _containing_ the top trends"

Reply:
ok